client/src/views/home/HomePageHeader.vue

```diff
@@ -113,14 +119,26 @@ import { onMounted, onUnmounted, ref, inject, Ref } from 'vue';
 import { Env } from '@/services/environment';
 import UpdateAppModal from '@/views/about/UpdateAppModal.vue';
 import { APP_VERSION } from '@/services/environment';
+import { openSettingsModal } from '@/views/settings';
+import { HotkeyGroup, HotkeyManager, HotkeyManagerKey, Modifiers, Platforms } from '@/services/hotkeyManager';
+import AboutModal from '@/views/about/AboutModal.vue';
+import { ParsecAccount } from '@/parsec';
+import { navigateTo, Routes } from '@/router';
 
 const { isSmallDisplay, isLargeDisplay } = useWindowSize();
 const injectionProvider: InjectionProvider = inject(InjectionProviderKey)!;
 const eventDistributor = injectionProvider.getDefault().eventDistributor;
 let eventCbId: string | null = null;
 const updateAvailability: Ref<UpdateAvailabilityData | null> = ref(null);
+const hotkeyManager: HotkeyManager = inject(HotkeyManagerKey)!;
+
+let hotkeys: HotkeyGroup | null = null;
 
 onMounted(async () => {
+  hotkeys = hotkeyManager.newHotkeys();
+  hotkeys.add({ key: ',', modifiers: Modifiers.Ctrl, platforms: Platforms.Desktop, disableIfModal: true }, openSettingsModal);
```

Comment:
You should remove these 2 keybinds from `HomePage.vue` if you're adding them here